validation-test/stdlib/SequenceType.swift.gyb

```diff
@@ -617,6 +617,26 @@ SequenceTypeTests.test("contains/Predicate") {
 }
 
 //===----------------------------------------------------------------------===//
+// first()
+//===----------------------------------------------------------------------===//
+
+SequenceTypeTests.test("first/Predicate") {
```

Comment:
Please add this test to `stdlib/private/StdlibCollectionUnittest/CheckSequenceType.swift` instead, since `first(where:)` is a customization point and different sequences and collections can implement it differently.